cmd/utils/cmd.go

```diff
@@ -403,8 +410,8 @@ func ExportAppendChain(blockchain *core.BlockChain, fn string, first uint64, las
 
 // ExportHistory exports blockchain history into the specified directory,
 // following the Era format.
-func ExportHistory(bc *core.BlockChain, dir string, first, last, step uint64) error {
-	log.Info("Exporting blockchain history", "dir", dir)
+func ExportHistory(bc *core.BlockChain, dir string, first, last, step uint64, f ExportFormat) error {
```

Comment:
I would not use an enum to dictate the output format, this should be done via the method naming, e.g.

`ExportHistoryEra1(..)`
`ExportHistoryEraE(..)`